README.md

```diff
@@ -1,2 +1,89 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+
+**NOTE**: The procedure will come with downtime while the applications are scaled down for the volume migration.
+## Prerequisite
+This guide assumes you have created a Rook-Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1. We assume that you have healthy cluster i.e. `ceph status` shows `health: OK`
+2. Minimum ceph version should be Ceph-CSI supported. Check [Ceph-CSI](https://github.com/ceph/ceph-csi#ceph-csi-features-and-available-versions) supported versions.
+3. Stop the application pods that are consuming the flex volume(s) that need to be converted.
+4. Ceph-CSI storageClass should be created in which you want to migrate.
+
+## Steps to Migrate
+1. Stop the application pods that are consuming the flex volume(s) that need to be converted.
+2. Create migrator pod `kubectl create -f manifests/migrator.yaml`
+3. Download the binary from release
```

Comment:
Show how to download and install?

Reply:
I'm not sure download link. Maybe once we have the release I'll update or if you have any suggestion I can add too